tests/conftest.py

```diff
@@ -52,12 +53,17 @@ def enable_filepath_feature(monkeypatch):
 
 
 @pytest.fixture(scope="session")
-def connection_root_bare():
-    connection = dj.Connection(
+def db_creds_root() -> Dict:
```

Comment:
We need the value of `os.getenv("DJ_HOST")` in several test modules. Best to perform this once and inject it as a fixture. Making a note to replace the pattern `os.getenv("DJ_HOST")` with this fixture in other modules besides `test_privileges`.

Reply:
Another thing that we should move to fixtures during the "big clean up".

Reply:
Why can't we just import `CONN_INFO` (as done in #1133)